Langref polish (#2193)
slint-ui/slint: docs/langref/src/modules.md

# Modules

> Components declared in a `.slint` file can be shared with components in other
> `.slint` files, by means of exporting and importing them.

Reviewer: ```suggestion
Components declared in a `.slint` file can be used as elements in other
`.slint` files, by means of exporting and importing them.
```

> By default, everything declared in a `.slint` file is private. The `export`

Reviewer: ```suggestion
By default, every type declared in a `.slint` file is private. The `export`
```

export { Button }
```

> In the above example, `Button` is usable from other `.slint` files, but

Reviewer: ```suggestion
In the above example, `Button` is accessible from other `.slint` files, but
```
